content/en/blog/2022/k8s-otel-expose.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Exposing OpenTelemetry Collector for cross cluster communication
+linkTitle: Exposing OpenTelemetry Collector
```

Comment:
For both the title and linkTitle you should use an article before "OpenTelemetry Collector".  Either "an" or "the" will work.